include/swift/Serialization/ModuleFormat.h

```diff
@@ -1078,11 +1079,13 @@ namespace decls_block {
     IdentifierIDField, // name
     DeclContextIDField,// context decl
     TypeIDField, // interface type
-    BCFixed<1>,  // has argument type?
```

Comment:
If you keep this around, you won't need maybeReadParameterList(), you'll know if you're reading a parameter list or not